runtime/protocol/http/common/src/aws/smithy/kotlin/runtime/http/response/HttpCall.kt

```diff
@@ -10,40 +10,47 @@ import aws.smithy.kotlin.runtime.http.HttpBody
 import aws.smithy.kotlin.runtime.http.request.HttpRequest
 import aws.smithy.kotlin.runtime.time.Instant
 import kotlinx.coroutines.CompletableJob
+import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Job
 import kotlin.coroutines.CoroutineContext
 import kotlin.coroutines.EmptyCoroutineContext
 
 /**
  * A single request/response pair
+ *
+ * @param request the original complete request
+ * @param response the [HttpResponse] for the given [request]
+ * @param requestTime the time the request was made by the engine
+ * @param responseTime the time the response was received. This is a rough estimate of Time-to-first-header (TTFH) as
+ * reported by the engine.
+ * @param coroutineContext the call context
  */
-public data class HttpCall(
-    /**
-     * The original complete request
-     */
+public open class HttpCall(
```

Comment:
**Question**: I'm not sure this class now belongs in the `response` subpackage (if indeed it ever did). Since we're breaking this anyway, do you want to move it up to `http`?